src/appleseed-max-impl/appleseedrenderer/renderersettings.cpp

```diff
@@ -752,6 +889,63 @@ IOResult RendererSettings::load_system_settings(ILoad* iload)
             result = read<bool>(iload, &m_enable_embree);
             break;
 
+          case ChunkSettingsSystemTextureCacheSize:
+            result = read<long long>(iload, &m_texture_cache_size);
+            break;
+        }
+
+        if (result != IO_OK)
+            break;
+
+        result = iload->CloseChunk();
+        if (result != IO_OK)
+            break;
+    }
+
+    return result;
+}
+
+IOResult RendererSettings::load_postprocessing_settings(ILoad* iload)
+{
+    IOResult result = IO_OK;
+
+    while (true)
+    {
+        result = iload->OpenChunk();
+        if (result == IO_END)
+            return IO_OK;
+        if (result != IO_OK)
+            break;
+
+        switch (iload->CurChunkID())
+        {
+        case ChunkSettingsPostprocessingDenoiseMode:
```

Comment:
Half-indent `case`, then half-indent cases bodies (so they are fully indented with respect to the `switch`). Example:

    switch (condition)
    {
      case 1:
        foo();
        break;
    }

Fix here and in other functions.